aws-elastic-beanstalk/README.md

```diff
@@ -0,0 +1,34 @@
+Seekret
+=======
+An AWS elastic beanstalk deployment options.
+
+## Pre-requisites
+
+- Elastic beanstalk deployment over Amazon linux v2
```

Comment:
please describe here that this option assumes a single container per EC2 Instance deployment. 
Also mention, that this deployment should be done **for each** target container for traffic capturing and suggest adding it to CD for automation.

Reply:
added